graphql_api/types/query/query.py

```diff
@@ -50,11 +52,17 @@ def resolve_owner(_, info, username):
         if not user or not user.is_authenticated:
             raise UnauthorizedGuestAccess()
 
-    return get_owner(service, username)
+        # per product spec, if guestAccess is off for the environment, the current enterpriseUser
+        # must be "activated" in the given target owner (e.g., "codecov" org) in order to see things
+        target = await get_owner(service, username)
+        if user.ownerid not in target.plan_activated_users:
```

Comment:
you can use this helper function: `current_user_part_of_org`

Reply:
Oh nice thanks!
Looking over [that helper](https://github.com/codecov/codecov-api/blob/68ade431abc69558315b571aced42974d5f0f38d/codecov_auth/helpers.py#L27) - Are we diligent about updating an owner (say `suejung-sentry`)'s `organizations` when we adjust the other source of truth on the organization (say `codecov`)'s `plan_activated_users`?
From what I could tell the term `activated` relates to whether the ownerid in the `plan_activated_users` list, and I haven't dug yet into what the owner.organizations is, but I thought it had to do with `accounts`?

Reply:
sounds like the helper may not be equivalent or what's desired here in this chat

<img width="432" alt="Screenshot 2024-10-23 at 8 40 00 AM" src="https://github.com/user-attachments/assets/145acb38-924a-4c05-a0f0-2c315e3285e1">